### PR TITLE
[1js] Add new port

### PR DESCRIPTION
**Pull request by Pospelove**

### CODE REVIEW

File: ports/1js/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "1js",
```
Comment (FrankXie05) [inline ports/1js/vcpkg.json:2]: Should we consider a recognizable name? 🧐

**Review by FrankXie05 — CHANGES_REQUESTED** [review]

File: ports/1js/vcpkg.json (hunk 1 above)
Comment (Pospelove) [reply ports/1js/vcpkg.json:2]: hmm, not sure, let me think a bit
it's 1js originally. pronounced like one-js

Comment (Pospelove) [reply ports/1js/vcpkg.json:2]: what names would you consider? @FrankXie05

Comment (FrankXie05) [reply ports/1js/vcpkg.json:2]: The name should be able to express the function of the current port simply and clearly. :)

**Pull request closed without merging**